dockerImages/rabbitmq/entrypoint.sh

```diff
@@ -19,6 +19,12 @@ trap 'sigusr1' USR1
 echo "Add weathervane domain to resolv.conf" 
 perl /updateResolveConf.pl
 
+newuid=$((4096 + RANDOM))
+olduid=$(id -u rabbitmq)
+echo "Assigning random uid $newuid to rabbitmq user with uid $olduid"
+usermod -u $newuid rabbitmq
+find / -user $olduid  -exec chown -h $newuid {} \;
+
```

Comment:
Is it possible for this to conflict with an existing user id on the host (running docker)?

Reply:
That is why I am am adding 4096 to the random number.  The pre-existing UIDs are either low numbers (less than 1024) or high numbers (nobody == 65534).  Since RANDOM return random numbers from 1 to 32767, the uids that I am generating will go from 4097 to 36863.  There could still be a conflict with another randomly generated uid, but the impact of that should be minimal.